src/applications/personalization/profile/mocks/rating-info/index.js

```diff
@@ -0,0 +1,53 @@
+const success = {
+  serviceConnected40: {
+    data: {
+      id: '',
+      type: 'evss_disability_compensation_form_rating_info_responses',
+      attributes: {
+        userPercentOfDisability: 40,
+      },
+    },
+  },
+};
+
+const error = {
+  badRequestUpstreamServer500: {
+    errors: [
+      {
+        title: 'Bad Request',
+        detail: 'Received a bad request response from the upstream server',
+        code: '500',
+        source: 'EVSS::DisabilityCompensationForm::Service',
+        status: '500',
+        meta: {},
+      },
+    ],
+  },
+  badRequest403: {
+    errors: [
+      {
+        title: 'Not authorized',
+        detail: 'User cannot access this resource',
+        code: '403',
+        source: 'EVSS::DisabilityCompensationForm::Service',
```

Comment:
In case you aren't aware, EVSS endpoints are going to be migrated over to Lighthouse in the future.

Reply:
This is a good point. Although we are not evaluating that `source` text, but the code, but I will make a note to evaluate our mocks and make sure they are up to date as we migrate to different sources. Thanks @Mottie !